src/index.html

```diff
@@ -142,16 +142,21 @@ <h2 class="selected-recipe-name-js">Placeholder: Recipe Name</h2>
         </div>
       </div>
       <div class="recipe-details flex-row">
-        <div class="ingredients-cost-section">
-          <section>
-            <h3>recipe cost</h3>
-            <p class="cost-js">Placeholder: this element will be removed and replaced</p>
+        <section class="left-wrapper">
+          <section class="ingredients-cost-section">
+            <p class="ingredients-needed-js">Placeholder</p>
           </section>
-          <section class="ingredient-list-section-js">
-            <h3>ingredients</h3>
-            <p>Placeholder: this element will be removed and replaced</p>
-          </section>
-        </div>
+          <div class="ingredients-cost-section">
+            <section>
+              <h3>recipe cost</h3>
+              <p class="cost-js">Placeholder: this element will be removed and replaced</p>
+            </section>
+            <section class="ingredient-list-section-js">
+              <h3>ingredients</h3>
+              <p>Placeholder: this element will be removed and replaced</p>
+            </section>
+          </div>
+        </section>
```

Comment:
I think this it was a great idea to add all of the ingredients needed to the side of the recipe page!